includes/InputToEntity/MinimalItemInput.php

```diff
@@ -113,6 +130,43 @@ public function getItem( array $inputEntity, PropertyId $reconcileUrlProperty )
 		return [ $item, array_values( $otherItems ) ];
 	}
 
+	/**
+	 * @param string $propertyId
+	 * @return PropertyId|null
+	 * @throws PropertyLabelNotResolvedException
+	 */
+	public function getPropertyId( string $propertyId ): ?PropertyId {
+			try {
+				return new PropertyId( $propertyId );
+			} catch ( \InvalidArgumentException $exception ) {
+				$this->logger->debug( $propertyId . ' could not be parsed as a propertyId.' );
+			}
+
+		return $this->getPropertyIdByLabel( $propertyId );
+	}
+
+	/**
+	 * @param string $label
+	 * @return PropertyId|null
+	 * @throws PropertyLabelNotResolvedException
+	 */
+	public function getPropertyIdByLabel( string $label ): ?PropertyId {
+		$entityIds = $this->propertyLabelResolver->getPropertyIdsForLabels( [ $label ] );
+
+		if ( empty( $entityIds ) ) {
+			throw new PropertyLabelNotResolvedException( $label, 'en' );
```

Comment:
That’s a WikibaseClient exception, and it also uses a message key that only exists in client. If we want to depend on Repo only, I think we should throw a `ReconciliationException` here, with a message of our own.

Reply:
This seems resolved.